style.css

```diff
@@ -4,6 +4,7 @@
   min-height: 100vh;
   flex-direction: column;
   background-color: white;
+  background-image: url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to adjust the path to your background image to replicate the Figma design 👍 
![image](https://user-images.githubusercontent.com/102757126/212639006-4c7eae15-7433-4d98-b4de-e2697c56d2f2.png)